src/main/java/com/github/fppt/jedismock/operations/RO_zadd.java

```diff
@@ -0,0 +1,43 @@
+package com.github.fppt.jedismock.operations;
+
+
+import com.github.fppt.jedismock.server.Response;
+import com.github.fppt.jedismock.server.Slice;
+import com.github.fppt.jedismock.storage.RedisBase;
+import jdk.nashorn.internal.runtime.regexp.joni.exception.InternalException;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+import static com.github.fppt.jedismock.Utils.deserializeObject;
+import static com.github.fppt.jedismock.Utils.serializeObject;
+
+public class RO_zadd extends AbstractRedisOperation {
```

Comment:
Maybe take a look at the base implementation of [add](https://github.com/fppt/jedis-mock/blob/master/src/main/java/com/github/fppt/jedismock/operations/RO_add.java)

I think you might be able to extend on that and get rid of some of the duplicate code. You can look [here](https://github.com/fppt/jedis-mock/blob/master/src/main/java/com/github/fppt/jedismock/operations/RO_sadd.java) as an example.